docs/README.md

```diff
@@ -0,0 +1,26 @@
+Table of Contents
```

Comment:
@derberg can you educate me about why `File Templates` goes so far at the end, instead of right after the `Templates` and `Authoring Template` pages? In my mind, I am missing the connection and I think I must need more context. 😀

Reply:
My thoughts exactly😅

Reply:
Even in the original authoring.md, the section "File templates" came right after authoring-templates🤔

Reply:
yeah, it might be so called `opinionated` decision 😄 

`File Templates` for me are needed for the user that already understood the basics of writing a generator template and the user is ready to write a code generator, as `File Templates` functionality was created mainly for code generation use case, where you need to generate multiple files out of a single file